main.py

```
@@ -0,0 +1,56 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC We will use this notebook to call the classes and functions
+
+# COMMAND ----------
+
+import pandas as pd
```

Ruff

`pandas` imported but unused

Remove unused import: `pandas`

(F401)